README.md

```diff
@@ -1,2 +1,177 @@
+
+<details>
+<summary>Database</summary>
+  
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **HTML linters**
+- **css linters**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Live Server installed in VScode
+- Basic understanding of html and css
+
+### Setup
+
+Clone this repository to your desired folder:
+> git clone https://github.com/francis450/Hello-Microverse-Project
+
+
+### Install
+
+Install this project with:
+
+> npm install
+
+### Usage
+
+To run the project:
+> Click  the `Go Live` button on the bottom right part of visual studio code editor.
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## Author
+- GitHub: [@francis450](https://github.com/francis450)
+- Twitter: [@nkamandefrancis](https://twitter.com/nkamandefrancis)
+- LinkedIn: [Francis](https://linkedin.com/in/njoroge-francis)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- **A navigation bar**
+- **A datailed footer**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project make a suggestion for more features. Any feedback will be welcome.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse Inc for this README template, It has been really helpful on this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **What license is used on the project?**
+
+  - This project uses MIT license
+
+- **What specific linters are used on the project**
+
+  - 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
[OPTIONAL] Nice one adding the license, and also ensuring that the link works as appropriate, I think it'd be better if this phrase is removed from this section, wouldn't it? 🙂.